app/containers/ItunesAppContainer/PlayingSongBar/index.js

```diff
@@ -0,0 +1,78 @@
+/**
+ *
+ * PlayingSongBar
+ *
+ */
+
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components';
+
+import { PauseCircleOutlined } from '@ant-design/icons';
+import If from '@app/components/If/index';
```

Comment:
`import If from '@app/components/If'`

Reply:
import the antd icons that you need in app/icons.js and then export them

your application code must import icons from  `app/icons.js`